File: 13420.py
for i in range(int(input())):
    a, op, b, equal, c = input().split()
    a, b, c = int(a), int(b), int(c)

    if op == '+':
        if a + b == c:
            correct = True
        else:
            correct = False
    elif op == '-':
        if a - b == c:
            correct = True
        else:
            correct = False
    elif op == '*':
        if a * b == c:
            correct = True
        else:
            correct = False
    else:
        if a // b == c:
            correct = True
        else:
            correct = False

    if correct:
        print("correct")
    else:
        print("wrong answer")

File: 13420.cpp
#include <iostream>
using namespace std;

int main() {
	int T; cin >> T;

	for (int i = 0; i < T; i++) {
		long long a, b, c; char op, equal;
		cin >> a >> op >> b >> equal >> c;

		bool correct;
		switch (op) {
		case '+':
			if (a + b == c) correct = true;
			else correct = false;
			break;

		case '-':
			if (a - b == c) correct = true;
			else correct = false;
			break;

		case '*':
			if (a * b == c) correct = true;
			else correct = false;
			break;

		case '/':
			if (a / b == c) correct = true;
			else correct = false;
			break;
		}

		if (correct) cout << "correct\n";
		else cout << "wrong answer\n";
	}
}
